File: src/test/java/social/alone/server/auth/oauth2/user/TokenProviderTest.java
package social.alone.server.auth.oauth2.user;


import org.junit.Test;
import org.springframework.beans.factory.annotation.Autowired;
import social.alone.server.BaseIntegrateTest;

import static org.assertj.core.api.Assertions.assertThat;

public class TokenProviderTest extends BaseIntegrateTest {

    @Autowired
    private TokenProvider tokenProvider;

    @Test
    public void test() {
        String token = tokenProvider.createToken(createdUser);
        assertThat(token).isNotEmpty();
    }

}

File: src/test/kotlin/social/alone/server/auth/oauth2/user/TokenProviderTest.kt
package social.alone.server.auth.oauth2.user


import org.assertj.core.api.Assertions.assertThat
import org.junit.Test
import org.springframework.beans.factory.annotation.Autowired
import social.alone.server.BaseIntegrateTest

class TokenProviderTest : BaseIntegrateTest() {

    @Autowired
    override lateinit var tokenProvider: TokenProvider

    @Test
    fun test() {
        val token = tokenProvider.createToken(createdUser)
        assertThat(token).isNotEmpty()
    }

}